crates/context/src/lib.rs

```diff
@@ -344,13 +346,14 @@ impl ContextManager {
             eyre::bail!("non-absolute path")
         };
 
-        self.install_application(blob_id, uri.as_str().parse()?, version)
+        self.install_application(blob_id, uri.as_str().parse()?, version, Vec::new())
```

Comment:
let's take in `metadata` here as well

Reply:
Added, this also required change in CLI commands